libc/fuzzing/stdlib/CMakeLists.txt

```
@@ -6,6 +6,14 @@ add_libc_fuzzer(
     libc.src.stdlib.qsort
 )
 
+add_libc_fuzzer(
+  heap_sort_fuzz
+  SRCS
+    heap_sort_fuzz.cpp
+  DEPENDS
+    libc.src.stlib.heap_sort
```

https://github.com/llvm/llvm-project/blob/bb064535bd071c1bddaf55ff7fe283fc8d23c1fc/libc/src/stdlib/CMakeLists.txt#L260-L270

```suggestion
  DEPENDS
    libc.src.stdlib.qsort_util
```